src/applications/edu-benefits/0994/containers/IntroductionPage.jsx

```diff
@@ -85,13 +85,23 @@ export class IntroductionPage extends React.Component {
                       want to attend (optional)
                     </li>
                   </ul>
+                  <div>
+                    <h6>To be eligible for VET TEC, you need to be</h6>
+                  </div>
+                  <ul>
+                    <li>
+                      A Veteran with at least one day of unexpired education
+                      benefits, <strong>or</strong>
+                    </li>
+                    <li>
+                      A service member with 180 days of less left on active duty
```

Comment:
I think "of" should be "or". It is correct based on the story, but the story is wrong.